pkg/issuer/venafi/client/venaficlient.go

```diff
@@ -41,12 +41,13 @@ import (
 )
 
 const (
-	tppUsernameKey    = "username"
-	tppPasswordKey    = "password"
-	tppAccessTokenKey = "access-token"
-	// Setting ClientId & Scope statically for simplicity
-	tppClientId = "cert-manager.io"
-	tppScopes   = "certificate:manage"
+	tppUsernameKey     = "username"
+	tppPasswordKey     = "password"
+	tppAccessTokenKey  = "access-token"
+	tppClientIdKey     = "client-id"
+	defaultTppClientId = "cert-manager.io"
```

Comment:
nitpick (non-blocking): Normally we'd capitalise "TPP" and "ID" here in our usual style, but since there's already some established styles here (e.g. in `vcert.Config.ClientId`) I don't think it's anything worth blocking a PR over!

```suggestion
	tppClientIDKey     = "client-id"
	defaultTPPClientID = "cert-manager.io"
```